### PR TITLE
Feature/pagination component

### PR DESCRIPTION
**Pull request by tienviet99**

### CODE REVIEW

File: src/components/pagination/index.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import Button from "components/button";
+import { FaAngleLeft, FaAngleRight } from "react-icons/fa";
+import clsx from "clsx";
+
+import { usePagination, DOTS } from "./usePagination";
+
+const Pagination = (props: any) => {
```
Comment (locbui95) [inline src/components/pagination/index.tsx:7]: do not use any

Comment (tienviet99) [reply src/components/pagination/index.tsx]: Done

File: src/components/pagination/index.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  } = props;
+
+  const paginationRange: (string | number)[] | undefined = usePagination(
+    currentPage,
+    totalCount,
+    siblingCount,
+    pageSize
+  );
+
+  const onNext = () => {
```
Comment (locbui95) [inline src/components/pagination/index.tsx:31]: wrong naming convention

Comment (tienviet99) [reply src/components/pagination/index.tsx]: Done

**Review by locbui95 — APPROVED** [review]

**Pull request merged**